### DISCUSSION TITLE
How does chromebrew persist updates?

### DISCUSSION THREAD

**Discussion by Apacelus** [skycocker/chromebrew#6867] [General]
I have written a small [script](https://github.com/Fornball/chromeos-autofanctrl) to keep my fans quiet, but it gets deleted every update. How does chromebrew manage to not get deleted?

**Comment by satmandu** [comment]
ChromeOS (chromeOS?) does not start anything in crosh at boot. Also, it doesn't wipe anything in `/usr/local`. So as long as you put the script in there it should stay across reboots. (But you will of course have to manually invoke your script at every boot by opening up `crosh`  and then invoking the script...)
